doc/sphinx-guides/source/api/getting-started.rst

```diff
@@ -52,6 +52,20 @@ If you ever want to check an environment variable, you can "echo" it like this:
 
   echo $SERVER_URL
 
+With curl version 7.56.0 and higher, it is recommended to use --from-string with outer quote rather than -F flag without outer quote.
+
+For example, curl command parameter below might cause error such as ``warning: garbage at end of field specification: ,"categories":["Data"]}``.
+
+.. code-block:: bash
+
+  -F jsonData={\"description\":\"My description.\",\"categories\":[\"Data\"]}
+
+Instead, use --from-string with outer quote. See https://github.com/curl/curl/issues/2022
+
+.. code-block:: bash
+
+  --from-string 'jsonData={"description":"My description.","categories":["Data"]}'
```

Comment:
```suggestion
  --form-string 'jsonData={"description":"My description.","categories":["Data"]}'
```

Reply:
@pdurbin 
Right. Sorry. I totally misunderstood.
Now I fixed it!